### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by RanimHelamiah**

### CODE REVIEW

File: lib/assets/image/crypto/logo.png
Comment (AMhaish) [inline lib/assets/image/crypto/logo.png:1]: This image should not be uploaded here, you can use the same path we have
https://cdn.dconfig.com/globalresources/1132/Resources/logo/Showcase%20Horizontal%20copy.png
And store it inside a variable in dConfig, then render that variable in an img tag

File: lib/organisms/components/navbars/navbar6/index.tsx [hunk 1]
```diff
@@ -48,14 +48,14 @@ const Navbar = ({
         <Box {...row}>
           <Logo
             href="/"
-            logoSrc={data.logo}
+            logoSrc={LogoImageAlt.src}
```
Comment (AMhaish) [inline lib/organisms/components/navbars/navbar6/index.tsx:51]: Should be coming also from dConfig "data.white_logo"

File: lib/assets/image/crypto/logo.png
Comment (RanimHelamiah) [reply lib/assets/image/crypto/logo.png:1]: done

File: lib/organisms/components/navbars/navbar6/index.tsx (hunk 1 above)
Comment (RanimHelamiah) [reply lib/organisms/components/navbars/navbar6/index.tsx:51]: done

**Pull request merged**